### PR TITLE
Trending

### LINKED ISSUE #16: As a user, I want to sort the questions based on popularity, so that I can find the most popular questions available.

### PR DESCRIPTION
**Pull request by RobbeBryssinck**

### CODE REVIEW

File: Abracadabra/AbracadabraAPI/AbracadabraAPI/Services/UpdateTrendingService.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            using (var scope = _scopeFactory.CreateScope())
+            {
+                var context = scope.ServiceProvider.GetRequiredService<AbracadabraContext>();
+                foreach (var question in context.Questions)
+                {
+                    var score = question.Upvotes - question.Downvotes;
+
+                    var order = Math.Log10(Math.Max(Math.Abs(score), 1));
+
+                    //double sign1 = Math.Sign(order);
```
Comment (VoidBGS) [inline Abracadabra/AbracadabraAPI/AbracadabraAPI/Services/UpdateTrendingService.cs]: Can it be automated using Math.Sign instead of the if statements?

Comment (RobbeBryssinck) [reply Abracadabra/AbracadabraAPI/AbracadabraAPI/Services/UpdateTrendingService.cs]: Updated in the latest commit.

**Review by VoidBGS — APPROVED** [review]
First version looks good.

**Review by Snechar — APPROVED** [review]
Looks good.

**Pull request merged**